src/theme/components/text.tsx

```diff
@@ -5,7 +5,9 @@
 import { Text, TextProps as TextPropsOriginal } from 'rebass'
 import styled from 'styled-components/macro'
 
-const TextWrapper = styled(Text)<{ color: keyof string }>`
+const TextWrapper = styled(Text).withConfig({
+  shouldForwardProp: (prop) => prop !== 'color',
```

Comment:
nit: I think this becomes more maintainable using the Array.prototype.includes pattern showcased in the [docs](https://styled-components.com/docs/api#shouldforwardprop):

    shouldForwardProp: (prop) => !['color'].includes(prop)